### DISCUSSION TITLE
setFieldsValue问题咨询

### DISCUSSION THREAD

**Discussion by joraboy328** [umijs/umi#11231] [General]
想手动修改Input值并显示， 但Form.Item name属性如果是数组格式，不生效，请教一下

<Form form={form} name="myform" onFinish={onFinish}>
        <Form.Item name={["material","Id"]}  label="原料ID:"  className="lables">
                            <Input/>
         </Form.Item> 
</Form>

          form.setFieldsValue({
              // name:['material','Id'],
              // value:'xxxx'
              'material.Id':'xxxxx'
          })  
        }};

**Comment by joraboy328** [comment]
<Form.Item name={["material","Id"]}  label="原料ID:"  className="lables">

**Reply by fz6m** [reply]
`form.setFieldValue(['a', 'b'], 'value')`
